packages/common/src/store/pages/chat/slice.ts

```diff
@@ -467,6 +482,15 @@ const slice = createSlice({
         return
       }
 
+      // Update the chat metadata
```

Comment:
@rickyrombo this part was a bit iffy to me. My intuition is that we _do not_ want to update `recheck_permissions` to `false` on every `addMessage`, only when we're calling `addMessage` with a message sent from a different user (not current user).

Reply:
Maybe a comment explaining why we don't update in one go after checking the existing message?